tests/repo/tests.py

```diff
@@ -131,3 +131,40 @@ def test_delete(self, api):
         self._setup_detail(api)
         self.runner.run(['content-delivery-repo', 'delete', '1'])
         self.assertEqual(api.calls['content-delivery-repos/1'], [('DELETE', {})])
+
+    def test_clone(self, api):
+        api.add_endpoint('rpc/content-delivery-repos/clone', 'POST', [
+            {
+                "arch": "x86_64",
+                "content_category": "binary",
+                "content_format": "iso",
+                "id": x,
+                "name": "rhel-x86_64-workstation-7-shadow",
+                "product_id": x,
+                "release_id": "rhel-7.1",
+                "repo_family": "dist",
+                "service": "rhn",
+                "shadow": True,
+                "variant_uid": "Workstation"
+            }
+
+            for x in range(1, 30)
+        ])
+        self._setup_detail(api)
+        with self.expect_output('list.txt'):
+            self.runner.run(['content-delivery-repo', 'clone',
+                             '--release-id-from', 'rhel-7.1-updates',
+                             '--release-id-to', 'rhel-7.1'
```

Comment:
fix in https://github.com/product-definition-center/pdc-client/pull/85/commits/d3a39a99703fdf8cf925cf8b9f85394c15e86662